### PR TITLE
Update German translation

### PR DESCRIPTION
**Pull request by Da2424**
Hi,
this patch contains the following changes:

* Set many translations to finished
* Added many new translations (more are planned)
* Removed vanished/obsolete translations
* Improved some translations (typos, grammar)
* Added quotes for file names/paths

I hope you like it :)

### CODE REVIEW

File: avidemux_de.ts [hunk 1]
```diff
@@ -99,130 +102,136 @@ If you re encode video we should drop these informations,
 If you just copy the video without reencoding,
  you should keep them.
 Drop timing informations ?</source>
-        <translation type="unfinished"></translation>
```
Comment (eumagga0x2a) [inline avidemux_de.ts:102]: The code containing this translatable string is excluded from build, the effort to translate it is therefore wasted.

File: avidemux_de.ts [hunk 2]
```diff
@@ -99,130 +102,136 @@ If you re encode video we should drop these informations,
...
+Timing-Informationen löschen?</translation>
     </message>
 </context>
 <context>
     <name>CalculatorDialog</name>
     <message>
         <location filename="../ADM_userInterfaces/ADM_dialog/calculator.ui" line="+16"/>
         <source>Calculator</source>
-        <translation type="unfinished">Video-Kalkulator</translation>
+        <translation>Video-Kalkulator</translation>
```
Comment (eumagga0x2a) [inline avidemux_de.ts:119]: Same, dead code, no use in touching translation.

File: avidemux_de.ts [hunk 3]
```diff
@@ -99,130 +102,136 @@ If you re encode video we should drop these informations,
...
     </message>
     <message>
         <location line="+12"/>
         <source>Target</source>
-        <translation type="unfinished">Format und Größe des Ausgabedatei</translation>
+        <translation>Format und Größe des Ausgabedatei</translation>
```
Comment (eumagga0x2a) [inline avidemux_de.ts]: If touching translations of dead code is questionable, touching them and not fixing an occurrence of a wrong article makes even less sense, I guess.

File: avidemux_de.ts [hunk 4]
```diff
@@ -284,91 +293,63 @@ Drop timing informations ?</source>
...
-        <translation type="obsolete">&amp;Filter</translation>
-    </message>
 </context>
 <context>
     <name>DialogOcr</name>
     <message>
         <location filename="../ADM_userInterfaces/ADM_ocr/ocr.ui" line="+13"/>
         <source>Dialog</source>
-        <translation type="unfinished">Dialog</translation>
+        <translation>Dialog</translation>
```
Comment (eumagga0x2a) [inline avidemux_de.ts:307]: Dead code.

File: avidemux_de.ts [hunk 5]
```diff
@@ -381,37 +362,37 @@ Drop timing informations ?</source>
...
         <location line="-7"/>
         <source>Speed</source>
-        <translation type="unfinished"></translation>
+        <translation>Geschwindigkeit</translation>
     </message>
     <message>
         <location line="-14"/>
         <source>0 fps</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```
Comment (eumagga0x2a) [inline avidemux_de.ts:395]: I am not sure that it is safe to rely on empty string as translation to be always replaced by the source string. Especially marking such translations as finished makes me feel uncomfortable. I may be easily wrong here, a reference to the relevant place in the documentation is welcome.

**Review by eumagga0x2a — COMMENTED** [review]
Thank you, I've added a few comments.

File: avidemux_de.ts (hunk 1 above)
Comment (Da2424) [reply avidemux_de.ts:102]: You're right. In this case, I overlooked that. Will this code never be re-enabled? For this case, the translation would be already present :)

File: avidemux_de.ts (hunk 2 above)
Comment (Da2424) [reply avidemux_de.ts:119]: Will this code never be re-enabled? I have translated it for this case. In some other translation, this context was also translated (maybe when the code was still active)

File: avidemux_de.ts (hunk 3 above)
Comment (Da2424) [reply avidemux_de.ts]: Oops, you're right. Actually, I checked every translation before I set it to finished, but I must have overlooked it here.
I double-checked the changes and made a commit.

File: avidemux_de.ts (hunk 5 above)
Comment (Da2424) [reply avidemux_de.ts:395]: I think it should be safe:
https://doc.qt.io/qt-5/linguist-translators.html#selecting-string-to-translate
_No Translation: The string does not have a translation. Click the icon to accept the empty translation anyway_
The docs also says that if the source will be changed, it should be reset to "Not Accepted"

But of course, I can change it to “real” translations, if you feel safer with that :)

**Comment by eumagga0x2a** [comment]
Thank you, not letting this PR bitrot here any longer, not promising not to revert / alter changes I don't feel comfortable about.

**Comment by Da2424** [comment]
Thanks for merging! If you want to change something, absolutley no problem for me.

**Pull request merged**